doc/content/design/coverage/index.md

```diff
@@ -8,7 +8,7 @@ revision: 2
 
 We would like to add optional coverage profiling to existing [OCaml]
 projects in the context of [XenServer] and [XenAPI]. This article
-presents how we do it. 
+presents how we do it.
```

Comment:
Can you at least separate all these spurious space changes in another commit?

Reply:
Github provides a button at the top of the diff view to ignore whitespace changes, if they are bothering you